### PR TITLE
[#1682] Redesign final screen

### LINKED ISSUE #1682: Changes to final screen once a new request is complete
Part of #1526

More sharing, better cross-selling

- [x] Extract "You've just made a request" stuff to a partial (#2078)
- [x] Link up elements here (https://github.com/mysociety/alaveteli/pull/2147#discussion_r24252160)
- [x] Implement sharing
- [x] Replace $authority_name$ with actual authority name in sidebar

screen shot 2014-07-03 at 15 30 42

### PR DESCRIPTION
**Pull request by wrightmartin**
Fixes #1682

This adds a redesigned final screen which includes better cross-selling and 'what to do now' information.

### CODE REVIEW

File: app/views/request/_request_sent.html.erb [hunk 1]
```diff
@@ -1,19 +1,66 @@
...
+                'annotation</a> below telling people ' \
+                'about your writing.', :url => new_comment_url(:url_title => @info_request.url_title).html_safe) %>
+        </p>
+      </div>
+      <div class="request-sent-message__column-2">
+        <div class="what-next">
+          <h2><%= _("What next?") %></h2>
+          <ul class="what-next__list">
+            <li>
+              <a href="#"><%= _("Change how you'd like to be notified of changes " \
```
Comment (garethrees) [inline app/views/request/_request_sent.html.erb]: What is this meant to link to?

**Comment by garethrees** [comment]
Looks great other than the one inline comment.

File: app/views/request/_request_sent.html.erb (hunk 1 above)
Comment (wrightmartin) [inline app/views/request/_request_sent.html.erb]: Ah, it was on the wireframes but the functionality doesn't exist

Comment (garethrees) [inline app/views/request/_request_sent.html.erb]: Fixed in https://github.com/mysociety/alaveteli/commit/cd1786ab79df6e1e9f23a2482c2d64ded872aa1b

**Pull request closed without merging**